### PR TITLE
[skip ci] Build UM7 from dev-access-esm1.6 branch

### PR DESCRIPTION
**Pull request by harshula**
* Also update git tags for other packages

### CODE REVIEW

**Comment by harshula** [comment]
The most recent, relevant updates that are in the `update_cable3` branch will be added to main via this PR.

File: spack.yaml [hunk 1]
```diff
@@ -2,9 +2,14 @@
...
 # configuration settings.
+
+# Build with:
+# - UM7 from dev-access-esm1.6 branch
+# - MOM5 from development branch
+# - CICE4 from ESM1.5 and change to CICE5 once CICE5 SPR has been updated.
 spack:
   specs:
-    - access-esm1p6@git.2024.10.0
+    - access-esm1p6@git.dev_2024.12.0
```
Comment (CodeGat) [inline spack.yaml:12]: This name isn't consistent with the `CALVER.MINOR` versioning format that we have for the other models...is there a reason for that?

**Review by CodeGat — CHANGES_REQUESTED** [review]

File: spack.yaml (hunk 1 above)
Comment (harshula) [reply spack.yaml:12]: Yes, it's not to be used for an official release. Is there another way to show that it should not be used for an official release?

Comment (CodeGat) [reply spack.yaml:12]: Nah, thats fine

**Review by CodeGat — APPROVED** [review]

**Comment by CodeGat** [comment]
Don't forget to have `[skip ci]` in the merge message, @harshula !

**Pull request merged**